.github/workflows/main.yml

```diff
@@ -1,25 +1,81 @@
-name: Dokku CD
+name: main
 on:
   push:
     branches: [main]
 env:
-  DOKKU_APP_NAME: "outbreak"
-  DOKKU_HOST: "dokku-1.hurumap.org"
   DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://dokku@dokku-1.hurumap.org/outbreak"
   GIT_PUSH_FLAGS: "--force"
 jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - id: deploy
-        name: Deploy to dokku
-        uses: idoberko2/dokku-deploy-github-action@v1
-        with:
-          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
-          dokku-host: ${{ env.DOKKU_HOST }}
-          app-name: ${{ env.DOKKU_APP_NAME }}
-          remote-branch: ${{ env.DOKKU_REMOTE_BRANCH }}
-          git-push-flags: ${{ env.GIT_PUSH_FLAGS }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+
+      - name: Check if version is bumped
+        id: version-check
+        uses: EndBug/version-check@v1
```

Comment:
Not sure I fully understand how this `version-check` is working for us. 
Are we just building new images if the `package.json` shows a change?
And how are we reflecting this to the Dokku `Dockerfile`?

Reply:
package version is stored in [`package.json`](https://github.com/CodeForAfrica/outbreak.AFRICA/pull/143/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R102). [`version-check`](https://github.com/EndBug/version-check#usage) checks if that version has actually been changed. You can change the file without making a new version e.g. when you want to make 2 quick fixes before deploying, etc.

Once there is a new version is detected that's when a new docker image will be built using the new version as tag (docker tag === project version).

Changing this version is done manually by whoever needs to deploy to prod. (there are tools like `yarn version` to bump `package.json` version but we'll need to find one that will bump on `Dockerfile` as well).

Reply:
Aaah makes more sense now.